modules/ROOT/pages/database-administration/aliases/manage-aliases-standard-databases.adoc

```diff
@@ -0,0 +1,842 @@
+
+| Valid values
+| Integer.
+
+Negative values are allowed and result in unlimited pool.
+Value of `0` is not allowed.
+
+| Default value
+| link:{neo4j-docs-base-uri}/operations-manual/{page-version}/configuration/configuration-settings#config_dbms.routing.driver.connection.pool.max_size[dbms.routing.driver.connection.pool.max_size]
+
+|===
+
+.dbms.routing.driver.logging.level
+[width="100%", cols="1s, 4a"]
+|===
+
+| Description
+| Sets level for driver internal logging.
+
+| Valid values
+| org.neo4j.logging.Level.
+
+One of `DEBUG`, `INFO`, `WARN`, `ERROR`, or `NONE`.
+
+| Default value
+| link:{neo4j-docs-base-uri}/operations-manual/{page-version}/configuration/configuration-settings#config_dbms.routing.driver.logging.level[dbms.routing.driver.logging.level]
+
+|===
+
+[NOTE]
+====
+If a transaction modifies a database alias, other transactions concurrently executing against that alias may be aborted and rolled back for safety.
+This prevents issues such as a transaction executing against multiple target databases for the same alias.
+====
+
+.Query
+[source, cypher]
+----
+CREATE ALIAS `remote-with-driver-settings` FOR DATABASE `northwind-graph-2020`
+AT "neo4j+s://location:7687"
+USER alice
+PASSWORD 'example_secret'
+DRIVER {
+  connection_timeout: duration({minutes: 1}),
+  connection_pool_max_size: 10
+}
+----
+
+When a database alias pointing to a remote database has been created, its details can be shown with the `SHOW ALIASES FOR DATABASE` command.
+
+.Query
+[source, cypher]
+----
+SHOW ALIAS `remote-with-driver-settings` FOR DATABASE YIELD *
+----
+
+.Result
+[role="queryresult"]
+----
++---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| name                          | composite | database               | location | url                       | user    | driver                                                   | properties |
++---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| "remote-with-driver-settings" | NULL      | "northwind-graph-2020" | "remote" | "neo4j+s://location:7687" | "alice" | {connection_pool_max_size: 10, connection_timeout: PT1M} | {}         |
++---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+----
+
+
+==== Set properties for remote database aliases
+
+Just as the local database aliases, the remote database aliases can be given properties.
+These properties can then be used in queries with the xref::database-administration/composite-databases/querying-composite-databases.adoc#composite-databases-queries-built-in-functions[`graph.propertiesByName()` function].
```

Comment:
same here regarding the link target switching from the function itself to the querying composite section